containerz/containerz.proto

```diff
@@ -314,6 +311,12 @@ message ListContainerResponse {
   }
 
   Status status = 4;
+
+  // Metadata associated to this container.
+  map<string, string> labels = 5;
```

Comment:
Do we need to define something about what is expected to be returned here? Is this any metadata? Can I pack arbitrary things into it as the server?

(I'm always a bit skeptical of these 'unstructured' ways to add data to an API, since there's many different things one _can_ put into them, and it becomes unclear what one really expects to get back.)

Reply:
This is about associating the running container with some labels in the form of arbitrary key-values. So for example, commit the binary was built at or version of the application. We could be prescriptive about these keys but that could lead to adding a bunch of fields here.

Reply:
This makes sense -- can we (as discussed) just note that this should not change the behaviour of the RPCs so that we ensure that we don't end up with this being used to avoid adding new fields to the proto.